clients/tabby-chat-panel/package.json

```diff
@@ -42,17 +42,21 @@
     "start": "tsx src/index.ts",
     "test": "vitest",
     "test:once": "vitest run",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "build:vscode": "yarn && esbuild src/vscode.tsx --bundle --minify --outfile=../vscode/chat-panel/index.js --loader:.js=jsx",
```

Comment:
edit build.config.ts

Reply:
left a FIXME comment, because I have no idea how to edit the config file, I don't wanna spend time to check this document at the moment